lsst-epo/investigations-client
# [F] Formalize toaster setup

JIRA: https://jira.lsstcorp.org/browse/EPO-8937

## What this change does ##

Formalizes the toaster that was created for the save answers form using `react-hot-toast` to create a reusable API for future toasts

## Notes for reviewers ##

Client-only change.

Include an indication of how detailed a review you want on a 1-10 scale.
- 2

## Testing ##

- Visit a page with a save form, click "save", and observe either a success or error toast appear then disappear after a short duration.
- Repeat the same but click to close
- Try adding your own toast to a page: 
```tsx
import { toast } from 'react-hot-toast'
import Toaster from '@/components/layout/Toaster'

function Component() {

    return <><button onClick={() => toast("I'm a toast")}/><Toaster /></>
}
```

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
